packages/documentation/src/pages/frontend-support-dashboard/app.jsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import Layout from '../../layouts/Layout';
+
+import CommitsTable from './containers/CommitsTable';
+import { vetsWebsiteInfo, contentBuildInfo } from '../../definitions/constants';
+
+import 'web-components/dist/component-library/component-library.css';
+
+import { defineCustomElements } from 'web-components/loader';
+defineCustomElements();
+
+
+async function fetchDashboardData(repo) {
+  // https://dmitripavlutin.com/javascript-fetch-async-await/#5-parallel-fetch-requests
+  const [
+    devBuildTextResponse,
+    stagingBuildTextResponse,
+    prodBuildTextResponse,
+    commitsResponse,
+  ] = await Promise.all([
+    fetch(repo.devBuildText),
+    fetch(repo.stagingBuildText),
+    fetch(repo.prodBuildText),
+    // last 30 commits from vets-website
+    fetch(`https://api.github.com/repos/${repo.owner}/${repo.repo}/commits`),
+  ]);
+
+  const devBuildText = await devBuildTextResponse.text();
+  const stagingBuildText = await stagingBuildTextResponse.text();
+  const prodBuildText = await prodBuildTextResponse.text();
+  const commits = await commitsResponse.json();
+
+  const result = {
+    devBuildText,
+    stagingBuildText,
+    prodBuildText,
+    commits,
+  };
+  console.log(result); // eslint-disable-line no-console
+
+  return result;
+}
+
+export default function App({ location }) {
+  const [appsDevBuildText, setAppsDevBuildText] = useState('');
+  const [appsStagingBuildText, setAppsStagingBuildText] = useState('');
+  const [appsProdBuildText, setAppsProdBuildText] = useState('');
+  const [appsCommits, setAppsCommits] = useState([]);
+  const [contentDevBuildText, setContentDevBuildText] = useState('');
+  const [contentStagingBuildText, setContentStagingBuildText] = useState('');
+  const [contentProdBuildText, setContentProdBuildText] = useState('');
+  const [contentCommits, setContentCommits] = useState([]);
```

Comment:
With this many state variables, consider maybe refactoring to use `useReducer` at some point.

Reply:
that's a great idea